docs/apv.md

```diff
@@ -557,9 +557,9 @@ Onderstaande regels tellen alleen op Narcopolis
 3. Bij overtreding van lid 1 of lid 2 zal een straf worden gegeven van de 1e categorie.
 4. Er mag op binnenkomende vluchten en schepen geschoten worden, die niet afkomstig zijn van de koninklijke marechaussee.
 
-### Artikel N-2 - Landen van helikopers/vliegtuigen
+### Artikel N-2 - Inladen van helikopers/vliegtuigen
```

Comment:
Waar slaat lid 2 dan nu op? je kan niet zomaar een naam wan een artikel aan passen en maar de helft van de inhoud.

Reply:
Aangezien de inhoud vrijwel hetzelfde blijft, dit is tevens ook besproken bij het staffoverleg.

Reply:
De strekking van de inhoud is zeker anders, en nu is lid 2 nog de oude strekking. ik zou verwachten namelijk dat lid 2 ook zou gaan over inladen maar het gaat over landen